.ci/openshift_e2e.sh

```diff
@@ -18,17 +18,45 @@ set -u
 # print each command before executing it
 set -x
 
-trap 'Catch_Finish $?' EXIT SIGINT
+function bumpPodsInfo() {
+    NS=$1
+    TARGET_FLD=${ARTIFACT_DIR}/$NS-info
```

Comment:
```suggestion
    TARGET_FLD="${ARTIFACT_DIR}/${NS}-info"
```
Optional (for clarity)

Reply:
Also I still read `FLD` as `field` -- consider using `DIR`

Reply:
Thanks. applied